pkgs/by-name/jr/jrl-cmakemodules/package.nix

```
@@ -0,0 +1,27 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+}:
+
+stdenv.mkDerivation {
+  pname = "jrl-cmakemodules";
+  version = "unstable-2024-04-12";
```

```suggestion
  version = "0-unstable-2024-04-12";
```

https://github.com/NixOS/nixpkgs/tree/master/pkgs#package-naming

---

Ok, thanks !